.github/workflows/ci-main.yml

```diff
@@ -285,7 +285,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        cfg: [Debug, Release]
+        cfg: [] # All vcpkg builds require xz. Disabled while vcpkg and xz upstreams discuss recovery from the backdoor.
```

Comment:
Can't one-line it. This triggers an error message :(

"The workflow is not valid. .github/workflows/ci-main.yml (Line: 288, Col: 14): Matrix vector 'cfg' does not contain any values"